apps/vs-code-designer/src/app/templates/dotnet/DotnetTemplateProvider.ts

```diff
@@ -65,6 +66,18 @@ export class DotnetTemplateProvider extends TemplateProviderBase {
     return await this.parseTemplates(context, projKey);
   }
 
+  public async getNet8Templates(context: IActionContext): Promise<ITemplates> {
+    const projKey = await this.getProjKey(context);
+    const projectFilePath: string = getDotnetProjectTemplatePath(this.version, projKey);
+    const itemFilePath: string = getDotnetItemTemplatePath(this.version, projKey);
+    const projectTemplatesURL = dotNet8ProjectTemplatesURL;
+    const itemTemplatesURL = dotNet8ItemTemplatesURL;
+
+    await Promise.all([downloadFile(context, projectTemplatesURL, projectFilePath), downloadFile(context, itemTemplatesURL, itemFilePath)]);
```

Comment:
Should we add a try catch? What if the download of the file saves? How are we handling it?